src/main/resources/application.conf

```diff
@@ -62,6 +62,14 @@ etc-client {
     }
   }
 
+  mining {
+    tx-pool-size = 1000
+    ommers-pool-size = 30
+    block-cashe-size = 30
+    coin-base = "0011223344556677889900112233445566778899"
```

Comment:
Also, as mentioned in the previous, comment we need to be able to modify the coinbase via RPC. Will having this as a config setting not interfere with its mutability?